src/modules/app_ruby/Makefile

```diff
@@ -30,7 +30,7 @@ ifneq (,$(findstring darwin,$(OS)))
 	LIBS += -L/opt/local/lib -L$(LOCALBASE)/lib -lev
 else
 	DEFS += -I$(LOCALBASE)/include -I$(SYSBASE)/include
-	LIBS += -L$(LOCALBASE)/lib -L$(SYSBASE)/lib -lruby -lpthread -ldl -lobjc
```

Comment:
@sergey-safarov you are changing the module here!!

Reply:
Yes Victor @linuxmaniac,
I made this because `objc` lib cannot find. I have created PR on Fedora29 environment. Think same error exist on CentOS.
I can see Travis-CI also build kamailio properly compiled without this options.
Is option really required?
What we can do to fix error with this options on RPM based dists?

Reply:
I don't know if is needed or not, I'm saying the changes need to be split. You are changing the app_ruby, those changes need to be in another commit with the proper commit message prefix

Reply:
Ok will do.